qa/TL0_python-self-test/test.sh

```diff
@@ -11,3 +11,7 @@ popd
 pushd ../TL0_python-self-test-operators
 bash -e ./test.sh
 popd
+
+pushd ../TL0_python-self-test-operators_b
+bash -e ./test.sh
+popd
```

Comment:
```suggestion
pushd ../TL0_python-self-test-operators_1
bash -e ./test.sh
popd

pushd ../TL0_python-self-test-operators_2
bash -e ./test.sh
popd
```
or
```suggestion
pushd ../TL0_python-self-test-operators_a
bash -e ./test.sh
popd

pushd ../TL0_python-self-test-operators_b
bash -e ./test.sh
popd
```

Reply:
Changed to `operators_1` and `operators_2`.